Model/Sales/TrackTraceHolder.php

```diff
@@ -236,6 +238,50 @@ private function checkLargeFormat(): bool
         return self::$defaultOptions->getDefaultLargeFormat('large_format');
     }
 
+    /**
+     * @param Order\Shipment\Track $magentoTrack
+     * @param object               $address
+     *
+     * @return bool
+     * @throws LocalizedException
+     */
+    private function getAgeCheck($magentoTrack, $address): bool
+    {
+        if ($address->getCountryId() !== AbstractConsignment::CC_NL) {
+            return false;
+        }
+
+        $ageCheckOfProduct    = $this->getAgeCheckOfProduct($magentoTrack);
+        $ageCheckFromSettings = self::$defaultOptions->getDefaultOptionsWithoutPrice('age_check');
+
+        return $ageCheckOfProduct ?? $ageCheckFromSettings;
+    }
+
+    /**
+     * @param Order\Shipment\Track $magentoTrack
+     *
+     * @return bool
+     * @throws LocalizedException
+     */
+    private function getAgeCheckOfProduct($magentoTrack): ?bool
```

Comment:
misschien is `getAgeCheckFromProducts` een betere naam

Reply:
done